bazel/version/compute_version.bzl

```diff
@@ -39,8 +39,6 @@ def compute_version(ctx):
     if not ctx.attr._release_flag[BuildSettingInfo].value:
         pre_release = ctx.attr._pre_release_flag[BuildSettingInfo].value
         pre_release_numbers = {
-            "alpha": ctx.attr._alpha_number_flag[BuildSettingInfo].value,
-            "beta": ctx.attr._beta_number_flag[BuildSettingInfo].value,
             "rc": ctx.attr._rc_number_flag[BuildSettingInfo].value,
         }
```

Comment:
Maybe not here, but it might be worth further simplification here, versus retaining the map with one entry.